packages/core/src/shipping/strategies/stripe-upe/stripe-upe-shipping-strategy.ts

```diff
@@ -131,17 +138,20 @@ export default class StripeUPEShippingStrategy implements ShippingStrategy {
             appearance,
         });
 
-        const shipping = this._store.getState().shippingAddress.getShippingAddress();
+        const shipping = getShippingAddress();
         const stripeState =
-            shipping?.stateOrProvinceCode && shipping.countryCode
-                ? getStripeState(shipping.countryCode, shipping.stateOrProvinceCode)
+            shipping?.stateOrProvinceCode && shipping?.countryCode
+                ? getStripeState(shipping?.countryCode, shipping?.stateOrProvinceCode)
```

Comment:
Are we able to remove the question marks on L144?

Reply:
Of course, even I changed it since the code has opportunities for improvement, I hope you like it https://github.com/bigcommerce/checkout-sdk-js/pull/1681/commits/129e1d2881440ebe20c3233fcb650ce2e3f2bd6d